README.rst

```diff
@@ -616,6 +616,28 @@ will order the results by the distance to the point (-122.4862, 37.7694).
 We can also reverse the order of the results by passing ``order=desc``:
 ``/location/?point=-122.4862,37.7694&order=desc&format=json``
 
+Schema Generation
+=================
+
+Note: Schema generation support is available only for DRF >= 3.12
```

Comment:
```suggestion
Note: Schema generation support is available only for DRF >= 3.12.
```

Please pay attention to punctuation and formatting, also take a look at the generated text at https://github.com/dhaval-mehta/django-rest-framework-gis#schema-generation